CONTRIBUTING.md

```diff
@@ -9,6 +9,8 @@ The document covers the process for contributing to the articles and code sample
 * [Building the docs](#building-the-docs)
 * [Contributor License Agreement](#contributor-license-agreement)
 
+This repository contains the conceptual documentation for .NET. There are separate repositories for the [code samples and snippets](https://github.com/dotnet/samples) and [API reference](https://github.com/dotnet/dotnet-api-docs). Issues and tasks for all three of those repositories are tracked here.
```

Comment:
Should we have a list instead of all the docs repos? We also have the roslyn-api-docs too. Are we tracking the work for that one here too?

Reply:
I added the Roslyn-api-docs repo both here and in the readme.